charts/openstack-cloud-controller-manager/templates/_helpers.tpl

```diff
@@ -43,6 +43,9 @@ component: controllermanager
 {{- define "occm.controllermanager.labels" -}}
 {{ include "occm.controllermanager.matchLabels" . }}
 {{ include "occm.common.metaLabels" . }}
+{{- range $key, $val := .Values.podLabels }}
+{{ $key }}: {{ $val }}
+{{- end }}
```

Comment:
As far as I understand, the `include` keyword for helm templating only works for including templates (i.e. `occm.common.metaLabels` defined a few lines above) and not for using values. I found this explanation in the helm docs: https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-include-function
I tried a few ways of using `include` but failed to use it with build in functions (i.e. "toYaml") or to use it to include the values directly.

I opted for changing the syntax to a conditional block and using the builtin `toYaml`, which looks cleaner in my opinion.

Another idea would be to define our own generic `toYaml` template:
```
{{- define "toYaml" -}}
{{- if . }}
{{- toYaml . }}
{{- end }}
{{- end -}}
```

which would then work with include, i.e.:
`{{ include "toYaml" .Values.podLabels }}`

But in my opinion the overhead of recreating builtins like this might not be worth it.
Please let me know if the implementation I chose works for you, or if you'd like me to add the generic "toYaml" describe above.